### PR TITLE
feat: publish to scene pool

### LINKED ISSUE #513: Publish to scene pool

### PR DESCRIPTION
**Pull request by belohlavek**
Screen Shot 2019-05-31 at 09 43 11

Closes #513

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -12,6 +12,7 @@ ENV REACT_APP_LOCAL_STORAGE_KEY builder
 ENV REACT_APP_MANA_TOKEN_CONTRACT_ADDRESS 0x2a8fd99c19271f4f04b1b7b9c4f7cf264b626edb
 ENV REACT_APP_ROLLBAR_ACCESS_TOKEN 46e4b7e45c844b9ab81315c8b0919e99
 ENV REACT_APP_SEGMENT_API_KEY H21EgRI4eYwICDZf5uW6ek2BiykIR6wA
+ENV REACT_BUILDER_SERVER_URL asd
```
Comment (belohlavek) [inline Dockerfile]: do not forget to change this

File: package.json [hunk 2]
```diff
@@ -78,9 +78,11 @@
     "typescript": "3.2.2",
     "url-loader": "1.1.1",
     "uuid": "^3.3.2",
+    "webm-writer": "^0.2.2",
```
Comment (belohlavek) [inline package.json]: remove

File: src/components/Modals/DeployModal/DeployModal.tsx [hunk 3]
```diff
@@ -1,57 +1,188 @@
...
+    }
   }
 
-  handleSubmitEmail = async () => {
-    const { email } = this.state
-    const analytics = getAnalytics()
+  componentWillUnmount() {
+    this.setState(this.getBaseState())
+    this.isSubmitting = false
+    this.isSuccess = false
```
Comment (cazala) [inline src/components/Modals/DeployModal/DeployModal.tsx]: should we move this to `state` ?

File: src/modules/translation/languages/en.json [hunk 4]
```diff
@@ -18,10 +18,21 @@
...
     "description": "Coming soon, you’ll be able to publish scenes directly from the Builder to your LAND! Subscribe to the Builder mailing list, and we’ll let you know when the feature is ready.",
     "email_thanks": "Thanks! We'll stay in touch!",
-    "title": "Publish to your LAND"
+    "title": "Publish to your LAND",
+    "pool": {
+      "title": "Publish your Scene",
+      "subtitle": "Submit your work to our scene pool for a chance to get it published. No LAND required.",
+      "contact_information": "Contact information",
+      "error_ocurred": "An error ocurred",
+      "i_accept_the": "I agree to receive news from Decentraland",
```
Comment (cazala) [inline src/modules/translation/languages/en.json]: this being mandatory is scary

**Comment by belohlavek** [comment]
Missing translations

File: src/components/Modals/AdBlockModal/AdBlockModal.tsx [hunk 5]
```diff
@@ -0,0 +1,39 @@
+import * as React from 'react'
+import { Close, Button } from 'decentraland-ui'
+import Modal from 'decentraland-dapps/dist/containers/Modal'
+import { t } from 'decentraland-dapps/dist/modules/translation/utils'
+import { getAnalytics } from 'decentraland-dapps/dist/modules/analytics/utils'
+
+import { Props } from './AdBlockModal.types'
+import { Campaign } from 'modules/analytics/campaigns'
```
Comment (belohlavek) [inline src/components/Modals/AdBlockModal/AdBlockModal.tsx]: order

File: src/components/Preview/Preview.types.ts [hunk 6]
```diff
@@ -27,11 +26,11 @@ export type Editor = {
   resetCameraZoom: () => void
   setCameraPosition: (position: Vector3) => void
   selectGizmo: (gizmo: Gizmo) => void
-  selectEntity: (entityId: string) => void
+  selectEntity: (entityId?: string) => void
```
Comment (belohlavek) [inline src/components/Preview/Preview.types.ts]: not optional

File: src/lib/api.ts [hunk 7]
```diff
@@ -1,15 +1,23 @@
...
 export const ASSETS_URL = env.get('REACT_APP_ASSETS_URL', '')
 export const EMAIL_SERVER_URL = env.get('REACT_APP_EMAIL_SERVER_URL', '')
 export const DAR_URL = env.get('REACT_APP_DAR_URL', '')
+export const BUILDER_SERVER_URL = env.get('REACT_APP_BUILDER_SERVER_URL', '')
 
 export enum EMAIL_INTEREST {
   MOBILE = 'builder-app-mobile',
   TUTORIAL = 'builder-app-tutorial',
-  PUBLISH = 'builder-publish-preview'
+  PUBLISH = 'builder-publish-preview', // TODO: deprecate this
```
Comment (belohlavek) [inline src/lib/api.ts]: remove this?

File: src/components/Banners/HomePageBanner/HomePageBanner.tsx [hunk 8]
```diff
@@ -47,11 +46,7 @@ export default class HomePageBanner extends React.PureComponent<Props> {
                 id="banners.dapper_homepage"
                 values={{
                   br: <br />,
-                  cta: (
-                    <a href="https://dap.pr/dclinstallp" rel="noopener noreferrer" target="_blank" onClick={this.handleHomePageClick}>
-                      {t('banners.dapper_homepage_cta')}
-                    </a>
-                  )
+                  cta: <a onClick={this.handleHomePageClick}>{t('banners.dapper_homepage_cta')}</a>
```
Comment (belohlavek) [inline src/components/Banners/HomePageBanner/HomePageBanner.tsx]: https://decentraland.org/blog/announcements/dapper-announcement/

for mobile

**Pull request merged**